Celeste.Mod.mm/Mod/Everest/Everest.Relinker.cs

```diff
@@ -219,6 +221,15 @@ public static class Relinker {
                     try {
                         Assembly asm = Assembly.LoadFrom(cachedPath);
                         _RelinkedAssemblies.Add(asm);
+
+                        ModuleDefinition mod = ModuleDefinition.ReadModule(cachedPath);
+                        if (!_RelinkedModules.ContainsKey(mod.Assembly.Name.Name))
+                            _RelinkedModules.Add(mod.Assembly.Name.Name, mod);
+                        else {
+                            Logger.Log(LogLevel.Warn, "relinker", $"Encountered module name conflict loading assembly {meta} - {asmname} - {mod.Assembly.Name}");
```

Comment:
This warning should probably specify that it's a cached assembly, since the verbose message above may not be included in the log

Reply:
Fixed in latest commit ac4a1b4cc6d5a34395073950544dfd5e8abe5ab3